lib/theme/app_theme.dart

```diff
@@ -59,5 +59,26 @@ class AppTheme {
             fontWeight: FontWeight.w800,
           ),
         ),
+        inputDecorationTheme: InputDecorationTheme(
+          fillColor: Colors.white24,
+          filled: true,
+          hintStyle: TextStyle(
+            color: Colors.white.withOpacity(0.3),
+            fontSize: AppDimensions.textSize17,
+            fontWeight: FontWeight.w400,
+          ),
+          contentPadding: const EdgeInsets.symmetric(
+            vertical: AppDimensions.spacing18,
+            horizontal: AppDimensions.spacing12,
+          ),
+          border: OutlineInputBorder(
+            borderSide: BorderSide.none,
+            borderRadius: BorderRadius.circular(AppDimensions.radius12),
+          ),
+        ),
+        textSelectionTheme: const TextSelectionThemeData(
+          selectionColor: Colors.white,
```

Comment:
```suggestion
          selectionHandleColor: Colors.white,
```

 **BEFORE**
<img width="105" alt="Screen Shot 2023-04-11 at 09 21 57" src="https://user-images.githubusercontent.com/18277915/231038939-d6292998-e07a-4e07-afd9-df790005cd8b.png">

**AFTER**
<img width="73" alt="Screen Shot 2023-04-11 at 09 20 58" src="https://user-images.githubusercontent.com/18277915/231038812-706844e8-a6b3-467f-99e5-d767cc7051d9.png">